modules/common/src/main/scala/surge/internal/streams/StreamManager.scala

```diff
@@ -107,7 +114,7 @@ class KafkaStreamManager[Key, Value](
 
   def getMetrics: Future[Map[MetricName, Metric]] = {
     implicit val timeout: Timeout = Timeout(metricFetchTimeout)
-    (managerActor ? KafkaStreamManagerActor.GetMetrics).mapTo[Map[MetricName, Metric]]
+    (managerActor ? KafkaStreamManagerActor.GetMetrics).mapTo[MetricsWrapper].map(_.metrics)(global)
```

Comment:
Does `handleGetMetrics` below in the actor also need to return a `MetricsWrapper` object? The `mapTo` here might figure it out, but if the actor still sends back a `Map` won't the serialization fail the same way?